crates/colorchoice-clap/examples/example.rs

```diff
@@ -0,0 +1,19 @@
+use anstream::println;
+use clap::Parser;
+use colorchoice_clap::Color;
+use owo_colors::OwoColorize as _;
+
+/// Foo
```

Comment:
We need an example because we want to let the users try it immediately. Both doc comments and examples are needed because they are targeted at different people.

These are the following commands I used to try out the example:

```
cargo run --example example
cargo run --example example -- --color=never
cargo run --example example -- > a.log
cargo run --example example -- --color=always > a.log
```

Reply:
I have gotten a lot of issues concerning the examples in https://github.com/console-rs/dialoguer lib. It led to me think that there is a category of users who would try out examples **and** read the docs before choosing the library.